projects/microphysics/experiments/dense-local-gscond.py

```diff
@@ -17,9 +17,13 @@
     "rnn-v1-shared-weights": rnn,
 }
 
+# It's a PITA to adjust transforms programmatically at this point time
```

Comment:
:(

Reply:
Haha, this was my concession after starting to write switches to preserve the T-scale vs. no T-scaling case with a single config file😩